internal/tests/pytests/wrappers/grid_stat/test_grid_stat_wrapper.py

```diff
@@ -689,6 +701,9 @@ def test_handle_climo_file_variables(metplus_config, config_overrides,
         ({'GRID_STAT_CENSOR_VAL': '12000, 5000', },
          {'METPLUS_CENSOR_VAL': 'censor_val = [12000, 5000];'}),
 
+        ({'GRID_STAT_SEEPS_P1_THRESH': 'gt4', },
+         {'METPLUS_SEEPS_P1_THRESH': 'seeps_p1_thresh = gt4;'}),
```

Comment:
```suggestion
         {'METPLUS_SEEPS_P1_THRESH': 'seeps_p1_thresh = ge0.1&&le0.85;'}),
```
Recommend using this more realistic P1 threshold instead, taken from this [MET unit test](https://github.com/dtcenter/MET/blob/80a9c1dd4755d50e8cb3b0b47811092eea7a1f82/internal/test_unit/xml/unit_point_stat.xml#L254).